admin_pages/general_settings/General_Settings_Admin_Page.core.php

```diff
@@ -563,6 +563,19 @@ protected function _update_admin_option_settings()
     /*************        Countries        *************/
 
 
+    /**
+     * @return string
+     */
+    protected function getCountryIsoForSite()
+    {
+        return isset(EE_Registry::instance()->CFG->organization->CNT_ISO)
```

Comment:
No need to use `isset` because `empty` check includes that check.
```suggestion
        return ! empty(EE_Registry::instance()->CFG->organization->CNT_ISO)
```

Reply:
no idea where those `span class="pl-s1"` tags came from :/